devtools-ci-index.yaml

```diff
@@ -826,7 +835,9 @@
             echo 'Using Host' $CICO_hostname
             set -x
 
+            {dl_env_toolkit}
             env > jenkins-env
+            ./env-toolkit dump > jenkins-env.json
             cat $creds_config_file >> jenkins-env
```

Comment:
wont you need to handle these as well ?

Reply:
I'm not sure I understand?

The files `env-toolkit` and `jenkins-env.json` are rsync'd to the target cico node. In the cico build scripts it can be loaded like this:

```bash
eval "$(./env-toolkit load -f jenkins-env.json DEVSHIFT_TAG_LEN QUAY_USERNAME QUAY_PASSWORD ...)"
```

Reply:
ahh sorry, I see what you mean.

Yes, we should, you are right.